packages/text-editor/src/provider/storage.ts

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import core, {
+  type Blob,
+  type BlobLookup,
+  type CollaborativeDoc,
+  type Ref,
+  collaborativeDocParse
+} from '@hcengineering/core'
+import { getClient } from '@hcengineering/presentation'
+import { ObservableV2 as Observable } from 'lib0/observable'
+import { applyUpdate, type Doc as YDoc } from 'yjs'
+
+interface EVENTS {
+  synced: (...args: any[]) => void
+}
+
+async function fetchContent (_id: Ref<Blob>, doc: YDoc): Promise<boolean> {
+  const client = getClient()
+
+  const blob = await client.findOne(core.class.Blob, { _id })
+  if (blob !== undefined) {
+    const update = await fetchBlobContent(blob as BlobLookup)
+    if (update !== undefined) {
+      applyUpdate(doc, update)
+      return true
+    }
+  }
+
+  return false
+}
+
+async function fetchBlobContent (blob: BlobLookup): Promise<Uint8Array | undefined> {
+  try {
+    const res = await fetch(blob.downloadUrl)
```

Comment:
You need to call `getBlobHref(undefined, _id)`

Reply:
Done